ci-operator/config/openshift/kubevirt-csi-driver/openshift-kubevirt-csi-driver-release-4.15.yaml

```diff
@@ -30,36 +30,7 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-4.13
   to: kubevirt-csi-driver
-- build_args:
-  - name: ARCH
-    value: amd64
-  - name: ldflags
-  dockerfile_literal: |-
-    FROM src
-    COPY oc hypershift /usr/bin/
-    RUN mkdir -p /go/src/github.com/openshift && \
-        cd /go/src/github.com/openshift && \
-        git clone https://github.com/openshift/kubevirt-csi-driver && \
-        git clone https://github.com/openshift/hypershift.git && \
-        cd hypershift/ && \
-        go mod tidy && go mod vendor && \
-        make e2e && \
-        sed -i 's|global-pull-secret.json|/tmp/global-pull-secret.json|g' ./hack/ci/install-cnv.sh
-    WORKDIR /go/src/github.com/openshift/hypershift
-  from: src
-  inputs:
-    cli:
-      paths:
-      - destination_dir: .
-        source_path: /usr/bin/oc
-    hypershift-operator-4.13:
-      paths:
-      - destination_dir: .
-        source_path: /usr/bin/hypershift
-  to: hypershift-cli-e2e
 promotion:
-  excluded_images:
-  - hypershift-cli-e2e
```

Comment:
i think you need to change `./ci-operator/step-registry/storage/conf/csi-kubevirt/storage-conf-csi-kubevirt-ref.yaml` to not reference this image anymore.